VEHICLE/GetVehicleNumberPlateTextIndex.md

```diff
@@ -8,17 +8,26 @@ ns: VEHICLE
 int GET_VEHICLE_NUMBER_PLATE_TEXT_INDEX(Vehicle vehicle);
 ```
 
-```
-Returns the PlateType of a vehicle  
-Blue_on_White_1 = 3,  
-Blue_on_White_2 = 0,  
-Blue_on_White_3 = 4,  
-Yellow_on_Blue = 2,  
-Yellow_on_Black = 1,  
-North_Yankton = 5,  
-```
+```c
+enum eVehiclePlateIndicies {
+	SanAndreasCursive = 0,
+	SanAndreasBlack = 1,
+	SanAndreasBlue = 2,
+	SanAndreasPlain = 3,
+	SRExcept = 4,
+	NorthYankton = 5,
+	// All indicies below this require b3095
+	eCola = 6,
+	LasVenturas = 7,
+	LiberyCity = 8,
+	LSCarMeet = 9,
+	LSPanic = 10,
+	LSPounders = 11,
+	Sprunk = 12,
+}
```

Comment:
It could just be the view I have on mobile but it seems you may be missing the closing of the code block from line 11

Reply:
It is indeed, nice catch.